### ISSUE TITLE
Upgrade from 4.8.1 to 4.10.0 fails on non-windows machines

### ISSUE THREAD

**Issue by Ellosaur** [tbroadley/spellchecker-cli#86]
Our dependabot opened a PR to perform the titular upgrade however the CI checks running on Git Hub actions execute the following script (via yarn) which fails:

```bash
 spellchecker -f [!CHANGELOG]*.md ./**/*.md -d .spelling -l en-GB
/usr/bin/env: ‘node\r’: No such file or directory
```

Executing the same command on windows works fine:

```powershell
$ spellchecker -f [!CHANGELOG]*.md ./**/*.md -d .spelling -l en-GB
Spellchecking 2 files...

README.md: no issues found
./test/README.md: no issues found
```

**Comment by tbroadley** [comment]
Thank you for pointing this out! I've just published v4.10.1, which I believe fixes this issue. I tested the fix on Ubuntu through Windows Subsystem for Linux. Let me know if you're still having trouble.

**Comment by Ellosaur** [comment]
@tbroadley sorted thank you!
